addon/services/unified-event-handler.js

```diff
@@ -218,6 +224,8 @@ export default Ember.Service.extend(Ember.Evented, {
    * @return {Void}
    */
   triggerEvent(eventName) {
-    Ember.run.throttle(this, () => this.trigger(eventName), EVENT_INTERVAL);
+    const throttleId = Ember.run.throttle(this, () => this.trigger(eventName), EVENT_INTERVAL);
```

Comment:
I realize this is not a new issue, but this can never actually throttle because we are always passing an anonymous function here (this is precisely why ember-lifeline only allows the “string” method name form of run.throttle). We should change this to ensure that throttle is called with a stable function.

Reply:
Related to this specific change, we should splice the specific timer out of the _throttledEventTimers array to prevent the array from growing unbounded during the life of a long running application.

Reply:
Spoke offline. I'm going to do the splice change request, but leave the throttle fix alone for a future fix.